### ISSUE TITLE
Process raw XML configuration files into structured data

### ISSUE THREAD

**Issue by ian-r-rose** [cagov/caldata-mdsa-caltrans-pems#77]
XML configuration files for districts need to be parsed into structured parquet files for load to the data warehouse. Dependent on #75

**Comment by peacexerox** [comment]
* The DB5 (which I had been daily shipping and we inspected in -depth) turned out only store raw data on D7 only (the XML file we have seen) 
* For other 8 districts data (D3,D4,D5,..D12) they were going thru the same server, but did not store the raw data as D7 does. 
* Both D7 and other ones ship the raw data to the DB96, which is the source for clearinghouse. 
* DB96 apparently contains ETL conversion logic to structure the data into VDS30SEC data. 
 
Conclusion: 
* We used to hypothesize the full districts have their raw XML files on either DB5 or its sister machine DB6. It is not the case. 
* We need to ship the DB96 to the AWS side because it contains full districts data. 

@ZhenyuZhu-Caltrans @kengodleskidot  @ian-r-rose

**Comment by pingpingxiu-DOT-ca-gov** [comment]
* The DB5 (which I had been daily shipping and we inspected in -depth) turned out only store raw data on D7 only (the XML file we have seen) 
* For other 8 districts data (D3,D4,D5,..D12) they were going thru the same server, but did not store the raw data as D7 does. 
* Both D7 and other ones ship the raw data to the DB96, which is the source for clearinghouse. 
* DB96 apparently contains ETL conversion logic to structure the data into VDS30SEC oracle table. 
 
Conclusion: 
* We used to hypothesize the full districts have their raw XML files on either DB5 or its sister machine DB6. It is not the case. 
* We need to ship the DB96 to the AWS side because it contains full districts data. 

@ZhenyuZhu-Caltrans @kengodleskidot  @ian-r-rose
